am_i_exploitable.py

```diff
@@ -36,89 +36,154 @@
-    parser.add_argument('--cve_id', type=str, default='', help='Enter CVE name according to the following format:'
-                                                               'cve_<YEAR>_<NUMBER> '
-                                                               'Otherwise, all CVEs will be checked')
-    parser.add_argument('--describe', type=bool, default=True, help='A description of the vulnerability '
-                                                                    '(True by default)')
+    parser.add_argument('--vulnerability_identifier', type=str, default='', help='Enter vulnerability identifier '
+                                                                                 'according to the following format:\n'
+                                                                                 '`CVE-<YEAR>-<NUMBER>`\nVulnerability '
+                                                                                 'name (if exists in db)\n`all` - scan '
+                                                                                 'your system for all the '
+                                                                                 'vulnerabilities in the cves directory'
+                                                                                 '\nIf the argument is not set, a menu '
+                                                                                 'message will appear presenting the '
+                                                                                 'currently supported vulnerabilities'
+                                                                                 '\n')
+    parser.add_argument('--description', type=bool, default=True, help='A description of the vulnerability '
+                                                                       '(True by default)')
     parser.add_argument('--graph', type=bool, default=False, help='Graph which presents the security checks of the '
                                                                   'vulnerability')
     parser.add_argument('--debug', type=bool, default=False, help='An option to debug the program and see errors')
-    parser.add_argument('--container', type=bool, default=False, help='Specify if you run in container or not')
+    parser.add_argument('--container', type=bool, default=False, help='Scan only running containers on the host')
+    parser.add_argument('--container_name', type=str, default='', help='Scan only running containers names that '
+                                                                       'mentioned.\nNeeds to be seperated by commas '
+                                                                       'only')
     return parser.parse_args()
 
 
+def set_argparse_parameters():
+    args = arguments()
+    if check_parameter_value(args):
+        if str(args.description).lower() == 'true':
```

Comment:
This logic can be a separate function that gets the parameter and sets it.
Then you can just call it for each relevant parameter.